changelog.md

```diff
@@ -3,6 +3,8 @@
 
 ## Changes affecting backward compatibility
 
+- `-d:nimPreviewJsonutilsHoleyEnum` becomes the default, `jsonutils` now can serialize/deserialize
+  holey enums as regular enums (via `ord`) instead of as strings.
```

Comment:
But why? Holey enums have a string representation just like other enums.

Reply:
The issue is when the option `joptEnumOrd` is enabled, normal enums generate integers, but holey enums generate strings (without the define). The other options `joptEnumSymbol` and `joptEnumString` exist to generate strings.

Another issue still exists though, `joptEnumSymbol` doesn't give the right output for holey enums either (because `symbolName` is not implemented for them), it acts like `joptEnumString`.